docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -186,6 +186,14 @@ This release includes a series of features that build on our new usage analytics
 
 {{< docs-imagebox img="/img/docs/v70/presence_indicator.jpg" max-width="1024px" caption="Grafana Enterprise - Presence Indicator" >}}
 
+### SAML Role and Team Sync
+
+SAML support in Grafana Enterprise is improved by adding Role and Team Sync. Read more about how to use these features in the [documentation]({{< relref "../enterprise/saml.md#configure-team-sync" >}}).
+
+### Okta OAuth Team Sync
+
+Okta gets its own provider which adds support for Team Sync. Read more about using Okta in the [documentation]({{< relref "../auth/okta.md" >}}).
```

Comment:
```suggestion
Okta gets its own provider which adds support for Team Sync. Read the [Okta documentation]({{< relref "../auth/okta.md" >}}) to learn more about using the Okta provider.
```

possibly repeat that it's specifically for Okta w/ OAuth since Okta provides multiple ways to connect to third-party applications

```suggestion
Okta gets its own OAuth provider which adds support for Team Sync. Read the [Okta documentation]({{< relref "../auth/okta.md" >}}) to learn more about using the Okta provider.
```